dev/TEAL.md

```diff
@@ -376,6 +423,31 @@ Asset fields include `AssetHolding` and `AssetParam` fields that are used in `as
 | `app_global_del` | delete key A from a global state of the current application |
 | `asset_holding_get i` | read from account A and asset B holding field X (imm arg) => {0 or 1 (top), value} |
 | `asset_params_get i` | read from asset A params field X (imm arg) => {0 or 1 (top), value} |
+| `app_params_get i` | read from app A params field X (imm arg) => {0 or 1 (top), value} |
+| `log` | write bytes to log state of the current application |
+
+### Inner Transactions
+
+The following opcodes allow for "inner transactions". Inner
+transactions allow stateful applications to have many of the effects
+of a true top-level transaction, programatically.  However, they are
+different in significant ways.  The most important differences are
+that they are not signed, and do not appear in the block in the usual
+away. Instead, their effects are noted in metadata associated with the
+associated top-level application call transaction.  An inner
+transaction's `Sender` must be the SHA512_256 hash of the application
+ID (prefixed by "appID"), or an account that has been rekeyed to that
+hash.
+
+Currently, inner transactions may perform `pay`, `axfer`, `acfg`, and
```

Comment:
This is inconsistent with https://github.com/algorandfoundation/specs/pull/52/files#diff-0526e4ea7095cf647c9c11573330c442ef80acf60053cb29c26d9c3a57db9babR790 which limits to only `pay` and `axfer`.

Reply:
True.  axfer and afrz came in last minute.  ledger.md updated.